.github/workflows/backfill-workflow-job.yml

```
@@ -0,0 +1,24 @@
+name: Backfill workflow jobs in Rockset
+
+on:
+  schedule:
+    - cron: "30 */1 * * *"
```

could we get a comment here for what this cron is

---

also what’s the difference from: 
```suggestion
    - cron: "30 * * * *"
```